x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rules_table/rules_table_filtering.cy.ts

```diff
@@ -28,9 +28,7 @@ import {
 import { disableAutoRefresh } from '../../../../tasks/alerts_detection_rules';
 import { getNewRule } from '../../../../objects/rule';
 
-// TODO: https://github.com/elastic/kibana/issues/161540
-// Flaky in serverless tests
-describe('Rules table: filtering', { tags: ['@ess', '@serverless', '@skipInServerless'] }, () => {
+describe('Rules table: filtering', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
Despite it is not skipped anymore, in the [flaky runner build](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/3197#018ab39f-c02e-4281-ad23-6c08f6a99f74) your attached in the description, I don't see any tests running. All of them seem to have been skipped:

<img width="1094" alt="Screenshot 2023-09-21 at 10 46 16" src="https://github.com/elastic/kibana/assets/7359339/5239d840-85b4-4c62-a6b4-041404866f1d">

This looks like a bug in the parallel.ts script or in the way you scheduled this runner. E.g. if you specified this concrete file path as your spec pattern (in this case, please specify some glob instead and re-run the flaky runner).

Reply:
Good point, I relied on the CI and haven't checked the logs as specifying a full file path used to work (I did it many times). Right now only pattern would work. I've updated the link in description.

It also reveals a problem in CI. Maybe we should make build steps running no tests fail with an explicit error like `There are no tests to run` as it shouldn't happen and has to be fixed. We had an issue of running steps with no tests to run with parallel execution in CI but it basically shouldn't happen.

Reply:
> Good point, I relied on the CI and haven't checked the logs as specifying a full file path used to work (I did it many times). Right now only pattern would work. I've updated the link in description.

Great, now I see that two tests in that file passed 👍

Reply:
> Maybe we should make build steps running no tests fail with an explicit error like There are no tests to run as it shouldn't happen and has to be fixed.

@maximpn Please elaborate on why this is necessary. Let's say there's a small group of tests that we want to run as a separate job on CI. If all of them were intentionally skipped for whatever reason, failing this job in `main` would block all open PRs. This may not be what we want.

I think what we could do is check if the `specPattern` parameter is a glob/folder or a path to a concrete file. If it's a concrete file, let Cypress run it even if `specPattern === grepSpecPattern`. This would be a bugfix for the parallel.ts script that would allow us to run single specs in the flaky runner.